package.json

```diff
@@ -5,7 +5,7 @@
   "repository": "git@github.com:ministryofjustice/use-of-force.git",
   "license": "MIT",
   "scripts": {
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
It would be better to just leave this as it is and use `start:dev` for development.
`start` is used in production